Engine/Engine.cs

```diff
@@ -184,8 +184,8 @@ public void Run(AlgorithmNodePacket job, AlgorithmManager manager, string assemb
                 catch (Exception err)
                 {
                     Log.Error(err);
-                    var runtimeMessage = "Algorithm.Initialize() Error: " + err.Message + " Stack Trace: " + err.StackTrace;
-                    _algorithmHandlers.Results.RuntimeError(runtimeMessage, err.StackTrace);
+                    var runtimeMessage = "Algorithm.Initialize() Error: " + err.Message + " Stack Trace: " + err;
+                    _algorithmHandlers.Results.RuntimeError(runtimeMessage, err.ToString());
```

Comment:
this impacts what ends up in the database, I think we recently this is up a certain way for a reason, lemme check the history here.